components/NavigationDrawer/src/MDCBottomDrawerTransitionController.m

```diff
@@ -0,0 +1,132 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCBottomDrawerTransitionController.h"
+
+#import "MDCBottomDrawerPresentationController.h"
+
+static const NSTimeInterval kMDCBottomDrawerOpenAnimationDuration = 0.34;
+static const NSTimeInterval kMDCBottomDrawerCloseAnimationDuration = 0.3;
+static const CGFloat kMDCBottomDrawerOpenAnimationSpringDampingRatio = 0.85f;
+
+@implementation MDCBottomDrawerTransitionController
+
+#pragma mark UIViewControllerTransitioningDelegate
+
+- (nullable id<UIViewControllerAnimatedTransitioning>)
+    animationControllerForPresentedController:(UIViewController *)presented
+                         presentingController:(UIViewController *)presenting
+                             sourceController:(UIViewController *)source {
+  return self;
+}
+
+- (id<UIViewControllerAnimatedTransitioning>)animationControllerForDismissedController:
+    (UIViewController *)dismissed {
+  return self;
+}
+
+- (UIPresentationController *)
+    presentationControllerForPresentedViewController:(UIViewController *)presented
+                            presentingViewController:(UIViewController *)presenting
+                                sourceViewController:(UIViewController *)source {
+  BOOL voiceOverIsRunning = UIAccessibilityIsVoiceOverRunning();
```

Comment:
I understand this is in a follow up PR but should we guard against other accessibility features as well, or is it just voiceover that effects this?

Reply:
added switch control too, thanks